src/java/org/apache/cassandra/metrics/ChunkCacheMetrics.java

```diff
@@ -7,79 +8,75 @@
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
  *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
  */
 package org.apache.cassandra.metrics;
 
-import java.util.concurrent.TimeUnit;
 import javax.annotation.Nonnull;
 
-import com.codahale.metrics.Timer;
+import com.google.common.annotations.VisibleForTesting;
+
 import com.github.benmanes.caffeine.cache.stats.CacheStats;
 import com.github.benmanes.caffeine.cache.stats.StatsCounter;
 import org.apache.cassandra.cache.ChunkCache;
 
-import static org.apache.cassandra.metrics.CassandraMetricsRegistry.Metrics;
-
-/**
- * Metrics for {@code ICache}.
- */
-public class ChunkCacheMetrics extends CacheMetrics implements StatsCounter
+public interface ChunkCacheMetrics extends StatsCounter
 {
-    /** Latency of misses */
-    public final Timer missLatency;
-
-    /**
-     * Create metrics for the provided chunk cache.
-     *
-     * @param cache Chunk cache to measure metrics
-     */
-    public ChunkCacheMetrics(ChunkCache cache)
+    /** Set this property to true in order to switch to micrometer metrics */
+    boolean USE_MICROMETER = Boolean.getBoolean(System.getProperty("cassandra.use_micrometer_metrics", "false"));
```

Comment:
When I talked to Ekaterina about  adding new properties she suggested that we should be using `CassandraRelevantProperties`, e.g. https://github.com/datastax/cassandra/blob/ds-trunk/src/java/org/apache/cassandra/config/CassandraRelevantProperties.java#L187

Can you update this to do the same?